src/easy/mod.rs

```diff
@@ -11,6 +11,7 @@ mod form;
 mod handle;
 mod handler;
 mod list;
+mod mime;
```

Comment:
`mime` is kinda its own API that isn't specific to the easy API, so I would make `mime` a top-level module.

Reply:
I can see what you're saying about "kinda its own API".
However, mime API doesn't have a meaning without the easy API, quite similar to the form API. So basically, I've put the mime module in the same level as the form module.

I don't mind extracting it, just thought it is worth double checking this before doing so.

Your call though, I'll do what you ask :)